high-level-tests/load-testing/data/generate_mass_data.sql

```diff
@@ -0,0 +1,871 @@
+BEGIN;
+
+-----------------------------------------------------------------------------------------------------
+--				Recalage des séquences       --------------------------------------------------------------
+-----------------------------------------------------------------------------------------------------
+SELECT setval(pg_get_serial_sequence('users','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "users";
+SELECT setval(pg_get_serial_sequence('assessments','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "assessments";
+SELECT setval(pg_get_serial_sequence('organizations','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "organizations";
+SELECT setval(pg_get_serial_sequence('memberships','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "memberships";
+SELECT setval(pg_get_serial_sequence('target-profiles','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "target-profiles";
+SELECT setval(pg_get_serial_sequence('target-profiles_skills','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "target-profiles_skills";
+SELECT setval(pg_get_serial_sequence('campaigns','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "campaigns";
+SELECT setval(pg_get_serial_sequence('campaign-participations','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "campaign-participations";
+SELECT setval(pg_get_serial_sequence('answers','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "answers";
+SELECT setval(pg_get_serial_sequence('knowledge-elements','id'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "knowledge-elements";
```

Comment:
Quand j'exécute le script sur une base complètement vide (juste après `db:migrate`) j'ai cette erreur :
```
psql:./high-level-tests/load-testing/data/generate_mass_data.sql:280: ERROR:  currval of sequence "users_id_seq" is not yet defined in this session
```
Si j'enlève le troisième argument de `setval` (`, max("id") IS NOT null`) le problème disparaît (mais les `id` des tables vides commencent à 2… pas bien grave 🤷)

J'ai des doutes sur les utilisations de `currval` dans le script de toute façon… ça n'a pas l'air indispensable.

Reply:
Ayant ajouté un DROP/CREATE de base, j'ai pushé un COMMIT qui enlève les CURRVAL